package.json

```diff
@@ -21,35 +21,35 @@
         "umd-analyze": "webpack-bundle-analyzer ./build_artifacts/umd_stats.json"
     },
     "devDependencies": {
```

Comment:
Can we bump up versions not to major versions to avoid other side effects/dealing with breaking changes

Reply:
Yep, I've only updated the required ones for npm install to run successfully. Other packages are minor version updates.